src/java_bytecode/java_object_factory.cpp

```diff
@@ -858,8 +860,10 @@ void java_object_factoryt::gen_nondet_pointer_init(
 
   // Alternatively, if this is a void* we *must* initialise with null:
   // (This can currently happen for some cases of #exception_value)
+  // Also, if the String model is not loaded then we can only
```

Comment:
??? Even if we have an incomplete java.lang.String, we can supply `{ @class_identifier = "java.lang.String" }`